draft-ietf-oauth-device-flow.xml

```diff
@@ -415,33 +468,169 @@
+      <t>
+        Support for the device flow MAY be declared in the OAuth 2.0
+        Authorization Server Metadata <xref target="I-D.ietf-oauth-discovery"/>
+        with the following metadata:
+      </t>
+      <t>
+      <list style="hanging"> 
+        <t hangText="device_authorization_endpoint">
+          <vspace/>
+          OPTIONAL.
+          URL of the authorization server's device authorization endpoint 
+          defined in <xref target="AuthorizationRequest"/>.
+        </t>
+      </list>
+      </t>
+    </section>
+
+    <section anchor="Security" title="Security Considerations">
+        <section title="User Code Brute Forcing" anchor="UserCodeBruteForce">
+          <t>
+          Since the user code is typed by the user, the entropy is typically
+          less than would be used for the device code or other OAuth bearer token types.
+          It is therefore recommended that the server rate-limit user code
+          attempts. The user code SHOULD have enough entropy that when combined
+          with rate limiting makes a brute-force attack infeasible.
+          </t>
+        </section>
+        <section title="Device Trustworthiness" anchor="DeviceTrustworthiness">
+          <t>
+          Unlike other native application OAuth 2.0 flows, the device requesting the
+          authorization is not the same as the device that the user grants access
+          from. Thus, signals from the approving user's session and device are
+          not relevant to the trustworthiness of the client device.
+          </t>
+        </section>
+        <section title="Remote Phishing" anchor="RemotePhishing">
+          <t>
+          It is possible for the device flow to be initiated on a device in
+          an attacker's possession. For example, they might send an email
+          instructing the target user to visit the verification URL and
+          enter the user code. To mitigate such an attack it is RECOMMENDED
+          during the User Instruction (<xref target="UserInstruction"/>)
+          to inform the user that they are authorizing a device, and to confirm
+          that the device is in their possession.
+          </t>
+          <t>
+          The user code needs to have a long enough lifetime to be useable (allowing
+          the user to retrieve their secondary device, navigate to the verification URI, login, etc),
+          but should be sufficiently short to limit the
+          usability of a code obtained for phishing. This doesn't prevent a
+          phisher presenting a fresh token, particularly in the case they
+          are interacting with the user in real time, 
+          but it does limit the viability of codes sent over email or SMS.
+          </t> 
+        </section>
+        <section title="Non-confidential Clients" anchor="NonConfidentialClient">
+          <t>
+          Most device clients are incapable of being confidential clients, as secrets
```

Comment:
Are you talking about the devices being unable to be confidential clients? I am wondering whether this is indeed true for many of those devices. What makes you believe that this is true?